docs/developer-guide/release-process-and-cadence.md

```diff
@@ -1,49 +1,78 @@
 # Release Process And Cadence
 
-Argo CD is being developed using the following process:
+## Release Cycle
 
-* Maintainers commit to work on set of features and enhancements and create GitHub milestone to track the work.
-* We are trying to avoid delaying release and prefer moving the feature into the next release if we cannot complete it on time.
-* The new release is published every **3 months**.
-* Critical bug-fixes are cherry-picked into the release branch and delivered using patch releases as frequently as needed.
+### Schedule
 
-## Release Planning
+These are the upcoming releases dates:
 
-We are using GitHub milestones to perform release planning and tracking. Each release milestone includes two type of issues:
+| Release | Release Planning Meeting | Release Candidate 1   | General Availability | Release Champion                                      | Checklist                                                     |
```

Comment:
Would it make sense to have primary & secondary release owners? primary should be an approver and secondary can be a reviewer or above. maybe something to consider if there are too many tasks for one person to handle in a release cycle.

Reply:
I like this idea of having another person as a shadow in case the primary is not available or gets overwhelmed with too many tasks.

Reply:
@harikrongali could you open an issue or another PR to add that? I like the idea, but wanna get this PR merged. :-)